src/Components/CoinListComponent/EachCoin.js

```diff
@@ -32,6 +34,12 @@ function EachCoin(props) {
         return daysInAWeek[indexedDate.getDay()]
     })
 
+    const delayChart = () => {
+        setTimeout(() => {
+            return <Line data={data} options={options}/>
+        }, 1000);
+        return <Line data={data} options={options}/>
+    }
```

Comment:
why did you do this?

Reply:
find another way please mate

Reply:
I'm not Harpreet

Reply:
Oops, I forgot to get rid of that, that solution to the update error doesn't actually work, I'll get rid of that!